README.md

```diff
@@ -20,7 +20,7 @@ To setup Zabbix proxy:
 1. Initialize proxy configuration structure: `./init-config.sh`.
 2. Setup proxy configuration by running necessary setup scripts or manually setting up files
 3. Create proxy docker container: `./create-proxy.sh`
-4. Start proxy docker container: `docker-compose start`
+4. Start proxy docker container: `docker start <container name>`
```

Comment:
Otherwise great but I wonder this docker start command here .. if it is really docker start then please specify the container name here

Reply:
For some unknown reason with docker-compose start command, it overrides all imported variables in docker-compose.yml. But if you start containers one by one with docker start, it works.

Reply:
docker-compose start reads the yml file and tries to extrapolate the variables which are not present. Not sure if that actually breaks containers or if --no-recreate could be used during up to avoid the problem.